src/components/i18nTextarea.vue

```diff
@@ -212,9 +200,24 @@ export default {
 			}
 			this.addTab(lang)
 		},
+		"$store.state.languages": function(languages) {
+			for (const lang in languages) {
+				if (languages[lang]) {
+					this.addTab(lang)
+				}
+			}
+		},
+
 	},
 	created() {
 		this.state = this.value || {}
+		const languages = this.$store.state.languages
+		for (const lang in languages) {
+			if (languages[lang]) {
+				this.addTab(lang)
+			}
+		}
+		this.focusOnTabWithLanguage('fi')
```

Comment:
why fixed string value

Reply:
After component gets created, the first time it should by default focus Finnish language, where in we have 3 languages already there prepoluated

Reply:
adding this kind of hidden constant string which is not visible on the project level is a bad idea. It should come from some variable which can be changed in the future without touching this place

Reply:
Do you mean we should keep this in .env file or something like that?